pnpm-lock.yaml

```diff
@@ -20386,6 +20361,13 @@ packages:
       yocto-queue: 1.0.0
     dev: true
 
+  /p-limit@5.0.0:
```

Comment:
now we have like 4 copies of this lib for some reason

Reply:
4 different version :/

Reply:
do you think I should delete pnpm-lock and regen?

Reply:
nawww but maybe we should do another cleanup + remove duplicates run.